Sources/DesignKit/Views/AnimationView.swift

```diff
@@ -0,0 +1,37 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Lottie
+
+public struct AnimationView: View {
+    let animation: LottieAnimation
+
+    public init(_ animation: PocketAnimation) {
+        self.animation = LottieAnimation.named(animation.rawValue, bundle: .module, subdirectory: "Lottie")!
+    }
+
+    public var body: some View {
+        LottieView(animation: animation)
+            .playing(loopMode: .loop)
```

Comment:
Do _both_ animations loop? In my opinion, the end-of-feed animation should animate once when first visible, then when it disappears get reset, rather than constantly looping when one is scrolled to the bottom. At least, that's my interpretation of what happens with this code.